### PR TITLE
Fixes date validation in facility triage form

### LINKED ISSUE #4862: BUG : Facility Triage is not working
**Describe the bug**

When we are trying to select the calendar date on the facility triage page, it keeps on giving us an error  " Date can't be in future "

**To Reproduce**

https://care.coronasafe.in/facility/4adaaff2-c6b2-4530-a518-3fbe44752421/triage

**Expected behavior**

We should be able to select the present and past dates in the triage calendar for inputting the field

**Screenshots**

image

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Fixes #4862 

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@rithviknishad 
image

the bug has not resolved

**Comment by nihal467** [comment]
@rithviknishad 
image

the calendar is not working properly

File: src/Components/Facility/TriageForm.tsx [hunk 1]
```diff
@@ -330,23 +312,17 @@ export const TriageForm = (props: triageFormProps) => {
...
-                    </div>
-                  )}
+                <DateFormField
+                  required
+                  name="entry_date"
+                  label="Entry Date"
+                  value={state.form.entry_date}
+                  disableFuture
+                  onChange={handleFormFieldChange}
+                  position="CENTER"
```
Comment (rithviknishad) [inline src/Components/Facility/TriageForm.tsx]:
```suggestion
                  position="RIGHT"
```

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**